meta-ostro/recipes-soletta/soletta/soletta_%.bbappend

```diff
@@ -1,6 +1,6 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
 
-DEPENDS_append = " systemd"
+DEPENDS_append = " systemd libmicrohttpd"
```

Comment:
shouldn't be going directly to meta-soletta ? this dependency management, why to have bbappend if we can put changes directly to meta-solleta ?